src/main/java/Duke.java

```diff
@@ -1,10 +1,193 @@
+                        break;
+                    }
+
+                    //Printing result
+                    System.out.println("Added:");
+                    System.out.println(" [T][ ] " + description);
+                    taskList.addToDo(description);
+                    System.out.println("Now you have " + taskList.getSize() + " tasks in the list.");
+                    break;
+
+                case ("deadline"):
+                    printLine();
+                    int byPosition = 0;
+                    String dueDate;
+                    for (int i=0; i<inputList.length; i++){
+                        if (inputList[i].equals("by")) {
+                            byPosition = i;
+                            break;
+                        }
+                    }
+                    
+                    // Handle invalid input
+                    try {
+                        description = getStringFromList(inputList, 1, byPosition);
+                        dueDate = getStringFromList(inputList, byPosition+1, inputList.length);
+                    } catch (IllegalArgumentException e){
+                        System.out.println("Invalid input: task/date not given!");
+                        break;
+                    }
+
+                    //Printing result
+                    System.out.println("Added:");
+                    System.out.println(" [D][ ] " + description + " (by: " + dueDate + ")");
+                    taskList.addDeadline(description, dueDate);
+                    System.out.println("Now you have " + taskList.getSize() + " tasks in the list.");
+                    break;
+
+                case ("event"):
+                    printLine();
+                    int atPosition = 0;
+                    String dateTime;
+                    for (int i=0; i<inputList.length; i++){
+                        if (inputList[i].equals("at")){
+                            atPosition = i;
+                            break;
+                        }
+                    }
+
+                    //Handle invalid input
+                    try{
+                        description = getStringFromList(inputList, 1, atPosition);
+                        dateTime = getStringFromList(inputList, atPosition+1, inputList.length);
+                    } catch (IllegalArgumentException e){
+                        System.out.println("Invalid input: task/date not given!");
+                        break;
+                    }
+
+                    System.out.println("Added:");
+                    System.out.println(" [E][ ] " + description + " (at: " + dateTime + ")");
+                    taskList.addEvent(description, dateTime);
+                    System.out.println("Now you have " + taskList.getSize() + " tasks in the list.");
+                    break;
+                
+                case ("delete"):
+                    printLine();
+                    int tasknumber = Integer.parseInt(inputList[1]);
```

Comment:
You can consider naming the variable `taskNumber` instead, in adherence to good naming practices of applying camel case for your variable names